pfcp/handler/handler.go

```diff
@@ -14,7 +14,8 @@ import (
 )
 
 func HandlePfcpHeartbeatRequest(msg *pfcpUdp.Message) {
-	logger.PfcpLog.Warnf("PFCP Heartbeat Request handling is not implemented")
+	h := msg.PfcpMessage.Header
```

Comment:
Should SMF check some content of Request here such as Recovery Timestamp here?

Reply:
SMF no need check Recovery Timestamp,  UPF initiate heartbeat request, and UPF storage Recovery Timestamp During Association Request. UPF should check Recovery Timestamp of Heartbeat Response from SMF.